docs/app/js/app.js

```diff
@@ -287,6 +287,10 @@ function(COMPONENTS, DEMOS, PAGES, $location, $rootScope) {
       $scope.isSelected = function() {
         return controller.isSelected($scope.section);
       };
+
+      $scope.itemClicked = function() {
+        controller.navItemClicked = true;
```

Comment:
Instead of `navItemClicked`, I recommend that we call this variable `autoFocusContent`; since we do not care **how** the value was set to true, but rather **why** the variable is used.